jax/_src/scipy/stats/pareto.py

```diff
@@ -47,7 +49,13 @@ def logpdf(x: ArrayLike, b: ArrayLike, loc: ArrayLike = 0, scale: ArrayLike = 1)
     array of logpdf values.
 
   See Also:
+    :func:`jax.scipy.stats.pareto.logcdf`
```

Comment:
For better HTML rendering, these should be in a list (i.e. put `- ` in front of each).

Same for other functions below.